src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -265,7 +265,7 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
         swapToCategory(builder, "permissions");
 
         enableColonyProtection = defineBoolean(builder, "enablecolonyprotection", true);
-        turnOffExplosionsInColonies = defineBoolean(builder, "turnoffexplosionsincolonies", true);
+        turnOffExplosionsInColonies = defineInteger(builder, "turnoffexplosionsincolonies", 0, 0, 2);
```

Comment:
shouldn't the 2nd zero be a one?

Reply:
defineInteger takes arguments builder, name, default value, minimum value, max value.  The minimum value should always be equal to or lower than the default value.  But OrionDevelopment's right that this is better handled with the defineEnum, so swapping to that.